src/components/OverlayAndCancelButton/OverlayAndCancelButton.tsx

```diff
@@ -23,3 +25,38 @@ export function LoadingOverlayAndCancelButton(loading: boolean, pid: string) {
         </>
     );
 }
+
+//Overlay to successfully terminate processes for tools using pkexec
+export function LoadingOverlayAndCancelButtonPkexec(
```

Comment:
Just requires a doc string and inline commenting.

- [ ] Implement inline comments and docstring.

Reply:
Implemented inline comments and docstring for OverlayAndCancelButton